<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1282</width>
    <height>854</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0.52613, y1:1, x2:0.482955, y2:0, stop:0.409091 rgba(150, 219, 232, 255), stop:1 rgba(255, 255, 255, 255));
    border-style: outset;
    border-width: 2px;
    border-color: beige;
}
QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:0.52613, y1:1, x2:0.482955, y2:0, stop:0.107955 rgba(150, 219, 232, 255), stop:0.420455 rgba(179, 239, 196, 255), stop:0.863636 rgba(173, 227, 232, 255));
	font: 20pt &quot;Bahnschrift SemiCondensed&quot;;
    color: rgb(16, 164, 255);
}
QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_Menu">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_Campaign">
           <property name="minimumSize">
            <size>
             <width>235</width>
             <height>140</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift SemiLight</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Kampania</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_MultiPlayer_2">
           <property name="minimumSize">
            <size>
             <width>235</width>
             <height>140</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Button">
               <brush brushstyle="LinearGradientPattern">
                <gradient startx="0.526130000000000" starty="1.000000000000000" endx="0.482955000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                 <gradientstop position="0.409091000000000">
                  <color alpha="255">
                   <red>150</red>
                   <green>219</green>
                   <blue>232</blue>
                  </color>
                 </gradientstop>
                 <gradientstop position="1.000000000000000">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </gradientstop>
                </gradient>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="LinearGradientPattern">
                <gradient startx="0.526130000000000" starty="1.000000000000000" endx="0.482955000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                 <gradientstop position="0.409091000000000">
                  <color alpha="255">
                   <red>150</red>
                   <green>219</green>
                   <blue>232</blue>
                  </color>
                 </gradientstop>
                 <gradientstop position="1.000000000000000">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </gradientstop>
                </gradient>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="LinearGradientPattern">
                <gradient startx="0.526130000000000" starty="1.000000000000000" endx="0.482955000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                 <gradientstop position="0.409091000000000">
                  <color alpha="255">
                   <red>150</red>
                   <green>219</green>
                   <blue>232</blue>
                  </color>
                 </gradientstop>
                 <gradientstop position="1.000000000000000">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </gradientstop>
                </gradient>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Button">
               <brush brushstyle="LinearGradientPattern">
                <gradient startx="0.526130000000000" starty="1.000000000000000" endx="0.482955000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                 <gradientstop position="0.409091000000000">
                  <color alpha="255">
                   <red>150</red>
                   <green>219</green>
                   <blue>232</blue>
                  </color>
                 </gradientstop>
                 <gradientstop position="1.000000000000000">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </gradientstop>
                </gradient>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="LinearGradientPattern">
                <gradient startx="0.526130000000000" starty="1.000000000000000" endx="0.482955000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                 <gradientstop position="0.409091000000000">
                  <color alpha="255">
                   <red>150</red>
                   <green>219</green>
                   <blue>232</blue>
                  </color>
                 </gradientstop>
                 <gradientstop position="1.000000000000000">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </gradientstop>
                </gradient>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="LinearGradientPattern">
                <gradient startx="0.526130000000000" starty="1.000000000000000" endx="0.482955000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                 <gradientstop position="0.409091000000000">
                  <color alpha="255">
                   <red>150</red>
                   <green>219</green>
                   <blue>232</blue>
                  </color>
                 </gradientstop>
                 <gradientstop position="1.000000000000000">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </gradientstop>
                </gradient>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Button">
               <brush brushstyle="LinearGradientPattern">
                <gradient startx="0.526130000000000" starty="1.000000000000000" endx="0.482955000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                 <gradientstop position="0.409091000000000">
                  <color alpha="255">
                   <red>150</red>
                   <green>219</green>
                   <blue>232</blue>
                  </color>
                 </gradientstop>
                 <gradientstop position="1.000000000000000">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </gradientstop>
                </gradient>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="LinearGradientPattern">
                <gradient startx="0.526130000000000" starty="1.000000000000000" endx="0.482955000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                 <gradientstop position="0.409091000000000">
                  <color alpha="255">
                   <red>150</red>
                   <green>219</green>
                   <blue>232</blue>
                  </color>
                 </gradientstop>
                 <gradientstop position="1.000000000000000">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </gradientstop>
                </gradient>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="LinearGradientPattern">
                <gradient startx="0.526130000000000" starty="1.000000000000000" endx="0.482955000000000" endy="0.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
                 <gradientstop position="0.409091000000000">
                  <color alpha="255">
                   <red>150</red>
                   <green>219</green>
                   <blue>232</blue>
                  </color>
                 </gradientstop>
                 <gradientstop position="1.000000000000000">
                  <color alpha="255">
                   <red>255</red>
                   <green>255</green>
                   <blue>255</blue>
                  </color>
                 </gradientstop>
                </gradient>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift SemiLight</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Pokój gry</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_WorkShop">
           <property name="minimumSize">
            <size>
             <width>235</width>
             <height>140</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift SemiLight</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Warsztat</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Shop">
           <property name="minimumSize">
            <size>
             <width>235</width>
             <height>140</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Bahnschrift SemiLight</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Sklep</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QWidget" name="stackedWidgetPage1">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="1">
         <widget class="QPushButton" name="pushButton_About">
          <property name="text">
           <string>About</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_LoadGame">
          <property name="text">
           <string>Load Game</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>120</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="pushButton_Exit">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_MultiPlayer">
          <property name="text">
           <string>MultiPlayer</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_NewGame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>New Game</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="pushButton_Connect">
          <property name="text">
           <string>Wejdź do gry</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>600</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="4">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="lineEdit_Login">
          <property name="text">
           <string>Test</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="lineEdit_Password">
          <property name="text">
           <string>Test</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_LoginMessage">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="NewGameForm" name="stackedWidgetPage1">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="FormSetIcon" name="stackedWidgetPage3">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NewGameForm</class>
   <extends>QWidget</extends>
   <header>newgameform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormSetIcon</class>
   <extends>QWidget</extends>
   <header>formseticon.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
